src/libs/actions/App.js

```diff
@@ -210,10 +210,16 @@ function setUpPoliciesAndNavigate(session, currentPath) {
 
 function openProfile() {
     const oldTimezoneData = myPersonalDetails.timezone || {};
-    const newTimezoneData = {
-        automatic: lodashGet(oldTimezoneData, 'automatic', true),
-        selected: moment.tz.guess(true),
-    };
+    let newTimezoneData;
```

Comment:
Couldn't we just initialize `newTimezoneData` to be `oldTimezoneData` and remove the `else` condition below? 😮

Reply:
Myesss i do like this idea better 😅